app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def update_comment_counter
+    comment.increment!(:comments_counter)
```

Comment:
- [ ] We are updating the comments counter for **a post** and not on a `comment`. Comments belong to a **post**

`post.increment!(:comments_counter)`